editor/tests/org.fusesource.ide.projecttemplates.tests.integration/src/main/java/org/fusesource/ide/projecttemplates/tests/integration/wizards/FuseIntegrationProjectCreatorRunnableForOSESringBootIT.java

```diff
@@ -42,4 +44,9 @@ protected void additionalChecks(IProject project) {
 		assertThat(project.findMember("src/main/java/META-INF/MANIFEST.MF")).as("A bad Manifest has been generated").isNull();
 		assertThat(project.getFile(".settings/fusetooling/Deploy " + project.getName() + " on OpenShift.launch").getLocation().toFile()).exists();
 	}
+	
+	@Override
+	protected void additionalMavenAttributes(ILaunchConfigurationWorkingCopy workingCopy) {
+		workingCopy.setAttribute(MavenLaunchConstants.ATTR_DEBUG_OUTPUT, true);
```

Comment:
That will blow up the log file length a lot. Maybe we should consider controlling that via System properties instead of a fixed value so we can enable / disable it without further code changes?

Reply:
> That will blow up the log file length a lot.

Yes that's why I activated it only on the specific test requiring it

> Maybe we should consider controlling that via System properties instead of a fixed value so we can enable / disable it without further code changes?

I prefer to have it in the code instead of "somewhere not versioned".

I would like to launch this PR several times first before merging in case we are able to have mor einformation on the "unstable issue" before "polluting" master in any way

Reply:
here we go: reproduced at the first try: https://issues.jboss.org/secure/attachment/12424197/MavenLaunchOutput-FuseIntegrationProjectCreatorRunnableForOSESringBootIT-OSESpringBootProject-2.18.1.redhat-000015.txt